src/core/errorHandler.js

```diff
@@ -31,41 +36,57 @@ export class ErrorHandler {
 
 class ErrorHandlerComponent extends React.Component {
   static propTypes = {
+    autoRenderErrors: PropTypes.boolean,
```

Comment:
The eslint rule we're blocked on upgrading on ([require-default-props](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md)) is asking for the opposite I think. If it isn't required it should have a default.